start_servers.sh

```diff
@@ -22,6 +22,14 @@ if [ -z ${REDIS_PORT} ]
         redis_port=${REDIS_PORT}
 fi
 
+# Check REDIS_CONF_PATH env variable for the path to redis.conf
+if [ -z ${REDIS_CONF_PATH} ]
+    then
+        redis_conf=''
+    else
+        redis_conf=${REDIS_CONF_PATH}
+fi
+
```

Comment:
This can be simplified to just the assignment. Should still work the same regardless of the existence of the env var.